settings.gradle

```diff
@@ -1,2 +1,2 @@
 rootProject.name = 'jade-data-repo'
-
+include 'datarepo-client'
```

Comment:
I'm not so sure about this. Please checkout [Authoring Multi-Project Builds](https://docs.gradle.org/current/userguide/multi_project_builds.html)

These are currently setup as independent projects. At a minimum, this will increase build time.

Reply:
I added a step so that the subproject tasks are not built by default when the jade-data-repo is built, which should address the concern about build time. From the parent gradle.build, I also checked the datarepo-client `buildEnvironment`, `properties`, and environment variables, and saw that they were datarepo-client specific. The only variables that may be of concern are those inherited by the parent jade-data-repo `gradle.properties`. Is there a concern in inheriting those?

Reply:
Sounds good! Thank you for checking it out.